docs/documentation/cn/administrator-guide/load-data/delete-manual.md

```diff
@@ -0,0 +1,178 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+#Delete
+
+Delete不同于其他导入方式，它是一个同步过程。和Insert into相似，所有的Delete操作在Doris中是一个独立的导入作业，一般Delete语句需要指定表和分区以及删除的条件来涮选要删除的数据，并将会同时删除base表和rollup表的数据。
+
+##语法
+
+主要的Delete语法如下：
+
+```
+DELETE FROM table_name [PARTITION partition_name]
+        WHERE
+        column_name1 op value[ AND column_name2 op value ...];
```

Comment:
if support "in value list"? for example, where key1 in (1,2,3)

Reply:
Not supported now. I 'll consider to add this support later.
And I fixed this document to add some note about "in value list"